src/components/SearchResults/SearchResults.jsx

```diff
@@ -0,0 +1,27 @@
+import Booking from "../Booking/Booking";
+import "./SearchResults.scss";
+
+const SearchResults = (props) => {
+  const bookings = props.bookings;
+  return (
+    <table data-testid="search-results-component">
+      <thead>
+        <tr>
+          <th scope="col">Name</th>
+          <th scope="col">Email</th>
+          <th scope="col">Room</th>
+          <th scope="col">Check in</th>
+          <th scope="col">Check out</th>
+          <th scope="col">Stay for</th>
+        </tr>
+      </thead>
+      <tbody>
+        {bookings.map((booking) => (
+          <Booking key={booking.id} booking={booking} />
+        ))}
+      </tbody>
+    </table>
+  );
+};
+
+export default SearchResults;
```

Comment:
looks good!

Reply:
Hi Hadika,
I'm actually sticking to the original file structure to keep things as simple as possible for everyone to understand.
I do prefer a different approach, using another syntax for importing/exporting components, declaring variables, etc.
If you think it would be easier to understand with a different approach, please provide an example, and I'll do my best to follow it.
Thanks.